global.json

```diff
@@ -1,5 +1,5 @@
 {
   "sdk": {
-    "version": "3.1.100"
+    "version": "3.1.200"
```

Comment:
Are we upgrading .net core to 3.1.2?  Should we upgrade all the projects?

Reply:
See my comments below on this file